.eslintrc.js

```diff
@@ -37,8 +42,7 @@ module.exports = createConfig({
     {
       files: tsFiles,
       rules: {
-        '@typescript-eslint/no-confusing-void-expression': 'off', // not yet supported by CRA
-        'testing-library/no-await-sync-events': 'off', // not yet supported by CRA
+        '@typescript-eslint/non-nullable-type-assertion-style': 'off', // conflicts with @typescript-eslint/no-non-null-assertion
```

Comment:
I'll remove once this is resolved: https://github.com/ljosberinn/eslint-config-galex/issues/235